GUIDE.md

```diff
@@ -156,6 +156,33 @@ See the Nix notes in the appendix at the end of this guide.
 As of this writing, they reflect using Nix instead of Stack, so updates and
 reconsideration of this process is called for.
 
+### Arch Linux
+
+To install dependencies, run this command as `root`:
+
+    pacman -S git ghc cabal-install alex happy haddock postgresql
```

Comment:
Also, https://github.com/commercialhaskell/stack/wiki/Downloads#arch-linux has some other points, something about a library being needed… and some note about something else…